include/nbl/video/SPhysicalDeviceLimits.h

```diff
@@ -0,0 +1,497 @@
+    //VkBool32                             shaderRoundingModeRTEFloat32;
+    //VkBool32                             shaderRoundingModeRTEFloat64;
+    //VkBool32                             shaderRoundingModeRTZFloat16;
+    //VkBool32                             shaderRoundingModeRTZFloat32;
+    //VkBool32                             shaderRoundingModeRTZFloat64;
+    //uint32_t                             maxUpdateAfterBindDescriptorsInAllPools;
+    //VkBool32                             shaderUniformBufferArrayNonUniformIndexingNative;
+    //VkBool32                             shaderSampledImageArrayNonUniformIndexingNative;
+    //VkBool32                             shaderStorageBufferArrayNonUniformIndexingNative;
+    //VkBool32                             shaderStorageImageArrayNonUniformIndexingNative;
+    //VkBool32                             shaderInputAttachmentArrayNonUniformIndexingNative;
+    //VkBool32                             robustBufferAccessUpdateAfterBind;
+    //VkBool32                             quadDivergentImplicitLod;
+    //uint32_t                             maxPerStageDescriptorUpdateAfterBindSamplers;
+    //uint32_t                             maxPerStageDescriptorUpdateAfterBindUniformBuffers;
+    //uint32_t                             maxPerStageDescriptorUpdateAfterBindStorageBuffers;
+    //uint32_t                             maxPerStageDescriptorUpdateAfterBindSampledImages;
+    //uint32_t                             maxPerStageDescriptorUpdateAfterBindStorageImages;
+    //uint32_t                             maxPerStageDescriptorUpdateAfterBindInputAttachments;
+    //uint32_t                             maxPerStageUpdateAfterBindResources;
+    //uint32_t                             maxDescriptorSetUpdateAfterBindSamplers;
+    //uint32_t                             maxDescriptorSetUpdateAfterBindUniformBuffers;
+    //uint32_t                             maxDescriptorSetUpdateAfterBindUniformBuffersDynamic;
+    //uint32_t                             maxDescriptorSetUpdateAfterBindStorageBuffers;
+    //uint32_t                             maxDescriptorSetUpdateAfterBindStorageBuffersDynamic;
+    //uint32_t                             maxDescriptorSetUpdateAfterBindSampledImages;
+    //uint32_t                             maxDescriptorSetUpdateAfterBindStorageImages;
+    //uint32_t                             maxDescriptorSetUpdateAfterBindInputAttachments;
+    //VkResolveModeFlags                   supportedDepthResolveModes;
+    //VkResolveModeFlags                   supportedStencilResolveModes;
+    //VkBool32                             independentResolveNone;
+    //VkBool32                             independentResolve;
+    //VkBool32                             filterMinmaxSingleComponentFormats;
+    //VkBool32                             filterMinmaxImageComponentMapping;
+    //uint64_t                             maxTimelineSemaphoreValueDifference;
```

Comment:
I don't think we expose or want timeline semaphore currently

Reply:
GL and GLES also should report `false` for the min-max filter stuff